src/PcaPsf.cc

```diff
@@ -66,6 +66,12 @@ PTR(afw::detection::Psf) PcaPsf::clone() const {
     return std::make_shared<PcaPsf>(*this);
 }
 
+PTR(afw::detection::Psf) PcaPsf::resized(int width, int height) const {
+    PTR(afw::math::LinearCombinationKernel)
+    kern = std::static_pointer_cast<afw::math::LinearCombinationKernel>(getKernel()->resized(width, height));
```

Comment:
it returns a  `shared_ptr<lsst::afw::math::Kernel>`, but I did think these kinds of casts were automatic...
```
 src/PcaPsf.cc:49:9: note: candidate constructor not viable: no known conversion from 'shared_ptr<lsst::afw::math::Kernel>' to
      'shared_ptr<afw::math::LinearCombinationKernel>' for 1st argument
```